docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -346,47 +346,23 @@ Once this has been done, then the release is ready to be published live.
 ### Stage 2: Release publication
 
 Once a release has been generated, has its translations updated, changelog generated,
-and tested, the next step is to publish the release. This involves tagging,
-building an archive, and uploading to
-[www.silverstripe.org](http://www.silverstripe.org/software/download/) download page.
-
-Invoked by running `cow release:publish` in the format as below:
+and tested, the next step is to publish the release by tagging all modules
+in the release plan.
 
 `cow release:publish <version> [<recipe>] -vvv`
 
-E.g.
+Example on how to publish the installer:
 
 `cow release:publish 4.0.1 silverstripe/installer`
 
-This command has these options:
+Options:
 
 * `-vvv` to ensure all underlying commands are echoed
 * `--directory <directory>` to specify the folder to look for the project created in the prior step. As with
   above, it will be guessed if omitted. You can run this command in the `./release-<version>` directory and
   omit this option.
-* `--aws-profile <profile>` to specify the AWS profile name for uploading releases to s3. Check with
-  damian@silverstripe.com if you don't have an AWS key setup.
-* `--skip-archive-upload` to disable both "archive" and "upload". This is useful if doing a private release and
-  you don't want to upload this file to AWS.
-* `--skip-upload` to disable the "upload" command (but not archive)
-
-As with the `cow release` command, this step is broken down into the following
-subtasks which are invoked in sequence:
-
-* `release:tag` Each module will have the appropriate tag applied (except the theme). All tags are pushed up to origin
```

Comment:
We still need this!

Reply:
I figured that there's no point breaking down the command in the readme if there's only one subcommand now (`cow release:publish` runs `cow release:tag`, and nothing else).